test/molochv3.test.js

```diff
@@ -66,4 +73,38 @@ contract('MolochV3', async accounts => {
     address = await contract.getAddress(moduleId);
     assert.equal(address, "0x0000000000000000000000000000000000000000");
   });
+
+  const numberOfShares = web3.utils.toBN('1000000000000000');
+  const sharePrice = web3.utils.toBN(web3.utils.toWei("120", 'finney'));
+  const remaining = sharePrice.sub(web3.utils.toBN('50000000000000'))
+
+  async function prepareSmartContracts() {
+    let lib = await FlagHelperLib.new();
+    let bank = await BankContract.new();
+    await MemberContract.link("FlagHelper", lib.address);
+    await ProposalContract.link("FlagHelper", lib.address);
+    let member = await MemberContract.new();
+    let proposal = await ProposalContract.new();
+    let voting = await VotingContract.new();
+    let daoFactory = await DaoFactory.new(bank.address, member.address, proposal.address, voting.address);
+    await daoFactory.newDao();
+    let pastEvents = await daoFactory.getPastEvents();
+    let daoAddress = pastEvents[0].returnValues.dao;
+    let onboarding = await OnboardingContract.new(daoAddress, numberOfShares, sharePrice);
+    let dao = await ModuleRegistry.at(daoAddress);
+    console.log(dao);
+    return {daoFactory, onboarding, voting, proposal, dao};
+  }
+
+  it("should not be possible to join a DAO if the proposal applicant is not active", async () => {
```

Comment:
I fixed this test case that was broken because the proposal applicant was not active yet, so the proposal was rejected.